supervision/video/backends/base.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import annotations
+
+from collections.abc import Iterator
+from typing import Protocol, runtime_checkable
+
+import numpy as np
+
+from ..utils import VideoInfo
```

Comment:
This import does not follow import standards. Let's update it to `from supervision.video.core import VideoInfo`.